dev-java/ant-contrib/ant-contrib-1.0_beta6_pre20201123.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom pom.xml --download-uri https://github.com/cniweb/ant-contrib/archive/0228412be2ef648cfabc1d74416d3188755aff9b.tar.gz --slot 0 --keywords "~amd64 ~ppc64 ~x86" --ebuild ant-contrib-1.0_beta6_pre20201123.ebuild
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="ant-contrib:ant-contrib:1.0b6"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+MY_COMMIT="0228412be2ef648cfabc1d74416d3188755aff9b"
+DESCRIPTION="Ant-contrib tasks for Apache Ant"
+HOMEPAGE="http://ant-contrib.sourceforge.net:80"
+SRC_URI="https://github.com/cniweb/${PN}/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+# Common dependencies
+# POM: pom.xml
+# commons-httpclient:commons-httpclient:3.1 -> >=dev-java/commons-httpclient-3.1:3
+# org.apache.ant:ant:1.9.15 -> !!!groupId-not-found!!!
+# org.apache.bcel:bcel:5.1 -> >=dev-java/bcel-6.5.0:0
+# org.apache.ivy:ivy:2.5.0 -> >=dev-java/ant-ivy-2.5.0:2
+# org.jvnet.hudson:ivy:1.4.1 -> !!!groupId-not-found!!!
```

Comment:
you might try to ignore this dep and check whether not including it really causes any issue.